vuepress/docs/next/tutorials/devops/caching-and-clustering.md

```diff
@@ -191,7 +203,7 @@ spec:
   ingressPath: /entando-de-app
   environmentVariables:
     - name: REDIS_ADDRESS
-      value: [your redis URI. For example redis://localhost:6379]
+      value: YOUR redis URI. For example redis://localhost:6379
```

Comment:
Redis

what is a URI?

Reply:
https://stackoverflow.com/questions/176264/what-is-the-difference-between-a-uri-a-url-and-a-urn/1984225#1984225